server.js

```diff
@@ -0,0 +1,24 @@
+//Required express, nconf and log4js
+const express = require('express');
+const nconf = require('nconf');
+const log4js = require('log4js');
```

Comment:
Configuration setup for log4js and nconf should be in separate module and those should be consume here.  
I will do in next pr. Will separate out everything.